composer.json

```diff
@@ -17,33 +17,29 @@
             "homepage": "http://mwop.net/"
         }
     ],
-    "repositories": [
-        {
-            "type": "composer",
-            "url": "https://packages.zendframework.com"
-        }
-    ],
     "require": {
-        "php": ">=5.6",
-        "zendframework/zend-eventmanager": "^3.0",
-        "zendframework/zend-json": "^2.5.0|^3.0.0",
-        "zendframework/zend-loader": "^2.5.0",
-        "zendframework/zend-mvc": "^2.7.13",
-        "zendframework/zend-paginator": "^2.5.0",
-        "zendframework/zend-uri": "^2.5.0",
-        "zendframework/zend-view": "^2.11.4",
-        "zendframework/zend-stdlib": "^3.2.1",
-        "zendframework/zend-hydrator": "^2.4.2"
+        "container-interop/container-interop": "^1.2",
+        "laminas/laminas-eventmanager": "^3.0",
+        "laminas/laminas-http": "^2.11",
+        "laminas/laminas-hydrator": "^2.4.2",
+        "laminas/laminas-json": "^2.5.0|^3.0.0",
+        "laminas/laminas-loader": "^2.5.0",
+        "laminas/laminas-mvc": "^2.7.13",
+        "laminas/laminas-paginator": "^2.5.0",
+        "laminas/laminas-servicemanager": "^3.4",
+        "laminas/laminas-stdlib": "^3.2.1",
+        "laminas/laminas-uri": "^2.5.0",
+        "laminas/laminas-view": "^2.11.4",
+        "php": ">=7.1"
```

Comment:
Also bumping to PHP 7.1 as the minimum version, which will warrant a major version bump. 

Technically 7.1 isn't supported anymore (https://www.php.net/supported-versions.php), but 5.6 to 7.1 seems like a good bump.